t/create-withdrawal-data.2-of-3-legacy.decoded

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
instead of calling these `legacy` let's pin them to the bitcoind version. We'll surely have more "legacy" fixtures later if there are more breaking changes we wish to support.

Reply:
I'm not sure what you mean by "pin them to the bitcoind version". "Legacy" in this case refers not to the bitcoind version but to the lack of segwit. Can you suggest a good term to use instead of legacy? Perhaps "nonsegwit"?

Reply:
"nonsegwit" SGTM